src/Sarif.FunctionalTests/ConverterTestData/FxCop/FxCopReportAllMessageStates.xml.sarif

```diff
@@ -131,6 +136,7 @@
         },
         {
           "ruleId": "CA1824",
+          "message": {},
```

Comment:
Why emitting empty `message` object?

Reply:
No idea! Gone.

Reply:
Actually this was because of a null result.message. Added a conditional in the FxCop converter.